src/OneSignalApi.ts

```diff
@@ -17,10 +17,6 @@ export default class OneSignalApi {
     return OneSignalApiShared.updatePlayer(appId, playerId, options);
   }
 
-  static sendNotification(appId: string, playerIds: Array<string>, titles, contents, url, icon, data, buttons) {
-    return OneSignalApiShared.sendNotification(appId, playerIds, titles, contents, url, icon, data, buttons);
```

Comment:
We should also clean up sendNotification from the `OneSignalApiShared` class too.